flask/launcher/src/main/java/net/corda/flask/launcher/OS.java

```diff
@@ -0,0 +1,42 @@
+package net.corda.flask.launcher;
+
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum OS {
```

Comment:
the problem is the location of the jar cache, there is some boring logic in `net.corda.flask.launcher.JarCache#computeCacheDirectory` to get the most *native-flavoured* file system location

Reply:
Why not `${user.home}/.flask`? But again, we already seem to find Capsule's solution here to be acceptable.

Reply:
I'm not sure about how Windows would deal with `${user.home}/.flask` (for sure it wouldn't be an hidden folder)
For linux and OSX would probably be ok, even if on linux the convention is to use `~/.cache` for cache files

Reply:
> I'm not sure about how Windows would deal with `${user.home}/.flask` (for sure it wouldn't be an hidden folder)
> For linux and OSX would probably be ok, even if on linux the convention is to use `~/.cache` for cache files

IIRC the `.capsule` directory was not hidden on Windows, but that was OK. And I think the `~/.cache` folder is more of a Free Desktop convention than a Linux convention.

Reply:
I'll remove that logic and simply use `${user.home}/.flask` then

Reply:
So can you delete this `enum OS` now please? I suspect that Gradle's built-in [OperatingSystem](https://docs.gradle.org/5.6.4/javadoc/org/gradle/nativeplatform/platform/OperatingSystem.html) API might be enough for any other uses it may have.

Reply:
I can't use Gradle API since this is used in the launcher process, not within Gradle itself, but I'll remove it either way and stick to `${System.getProperty("user.home")}/.flask`